Client/Frontend/Onboarding/Models/IntroViewModel.swift

```diff
@@ -44,4 +58,15 @@ class IntroViewModel: OnboardingViewModelProtocol, FeatureFlaggable {
     func saveHasSeenOnboarding() {
         introScreenManager?.didSeeIntroScreen()
     }
+
+    // MARK: SkAdNetwork
+    // this event should be sent in the first 24h time window, if it's not sent the conversion value is locked by Apple
+    func sendOnboardingUserActivationEvent() {
+        let fineValue = OnboardingOptions.allCases.map { chosenOptions.contains($0) ? $0.rawValue : 0 }.reduce(0, +)
+        let conversionValue = ConversionValueUtil(fineValue: fineValue, coarseValue: .low, logger: DefaultLogger.shared)
+        // we should send this event only if an action has been selected during the onboarding flow
+        if fineValue > 0 {
+            conversionValue.adNetworkAttributionUpdateConversionInstallEvent()
```

Comment:
nit: We might want to rename the function `adNetworkAttributionUpdateConversionInstallEvent`, since we're using it for non-install events. `adNetworkAttributionUpdateConversionEvent` may be sufficient